kafka_consumer/tests/docker/kerberos/docker-compose.yaml

```diff
@@ -0,0 +1,132 @@
+version: '3.9'
+
```

Comment:
nit: These actually don't have an effect anymore and we should start removing them

```suggestion
```